src/Symfony/Bundle/FrameworkBundle/Routing/Loader/Configurator/Traits/RouteTrait.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Routing\Loader\Configurator\Traits;
+
+use Symfony\Component\Routing\Loader\Configurator\RouteConfigurator;
+
+trait RouteTrait
```

Comment:
what about naming this NoControllerTrait?

Reply:
Good call!

Reply:
Removed after our discussion on Slack.